airbyte-integrations/connectors/source-mailchimp/README.md

```diff
@@ -0,0 +1,62 @@
+# Mailchimp Source 
+
+This is the repository for the Mailchimp source connector, written in Python. 
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/sources/mailchimp).
+
+## Local development
+### Build
+First, build the module by running the following from the `airbyte` project root directory: 
+```
+./gradlew :airbyte-integrations:connectors:source-mailchimp:build
+```
+
+This should generate a virtualenv for this module in `source-mailchimp/.venv`. Make sure this venv is active in your
+development environment of choice. If you are on the terminal, run the following from the `source-mailchimp` directory:
+```
+cd airbyte-integrations/connectors/source-mailchimp # cd into the connector directory
+source .venv/bin/activate
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv. 
+
+**All the instructions below assume you have correctly activated the virtualenv.**.
+
+### Locally running the connector
+```
+python main_dev.py spec
+python main_dev.py check --config sample_files/test_config.json
+python main_dev.py discover --config sample_files/test_config.json
+python main_dev.py read --config sample_files/test_config.json --catalog sample_files/test_catalog.json
+```
+
+### Unit Tests
+To run unit tests locally, from the connector directory run:
+```
+pytest unit_tests
+```
+
+### Locally running the connector docker image
+```
+# in airbyte root directory
+./gradlew :airbyte-integrations:connectors:source-mailchimp:buildImage
```

Comment:
FYI anywhere it says `buildImage`/`buildTestImage` should be replaced with `airbyteDocker`/`airbyteDockerTest` respectively. See #901